cider/exec/template/DateTimeIR.cpp

```diff
@@ -273,11 +274,12 @@ std::unique_ptr<CodegenColValues> CodeGenerator::codegenExtract(
             get_extract_timestamp_precision_scale(extract_expr->get_field())));
   }
   const auto extract_fname = get_extract_function_name(extract_expr->get_field());
-
-  llvm::Value* res_lv =
-      cgen_state_->emitExternalCall(extract_fname,
-                                    get_int_type(64, cgen_state_->context_),
-                                    std::vector<llvm::Value*>{fromtime_lv});
+  std::vector<llvm::Value*> extract_lvs{fromtime_lv};
+  if (fromtime_nullable->getNull()) {
+    extract_lvs.push_back(fromtime_nullable->getNull());
```

Comment:
I think we can just call a function to check whether `fromtime_nullable->getNull()` is null. Or add else block 
```
else {
extract_lvs.push_back(llvm::Value(True);
}
```